test/unit/clients/client.test.ts

```diff
@@ -5,12 +5,15 @@ import * as github from "@actions/github";
 
 import Client from "../../../src/clients/client";
 
+type MockedOctokit = jest.MockedObjectDeep<ReturnType<typeof github.getOctokit>>; // eslint-disable-line max-len
+
 describe("clients/client.ts", () => {
-  let client;
-  let octokit;
+  let client: InstanceType<typeof Client>;
+  let octokit: MockedOctokit;
 
   beforeAll(() => {
-    octokit = github.getOctokit("token");
+    const _octokit = github.getOctokit("token");
+    octokit = _octokit as MockedOctokit;
```

Comment:
This, and similar changes, can be simplified to

```suggestion
    octokit = github.getOctokit("token") as MockedOctokit;
```